horace_core/admin/cache.py

```diff
@@ -82,13 +81,11 @@ def read_chunk(filename,start,size,buf_size,progress,n_workers):
     if progress:
         prog_rep = progress_rep(size,n_workers)
         
-    fh = open(filename,'rb')
-    if fh<0:
-        raise ValueError("Can not open file: "+filename)
+    fh = open(filename,'rb',buffering=0)
```

Comment:
The other form (`with`) is the standard way of doing things.

Reply:
yes, and I am using it in when reading chunk-of-file but for single seek using it looks confusing.

Reply:
You do not need the seek and this file is only coincidentally closed which is very confusing